wall/wall.c

```diff
@@ -70,14 +69,15 @@ void main(int argc, char *argv[]) {
 	strncpy(sterm, ttyname(fileno(stderr)), sizeof(sterm));
 	for (i = 1; sterm[i] != '/'; i++);
 	strncpy(sterm, &sterm[(i + 1)], UT_LINESIZE);
+	sterm[(UT_LINESIZE + 1)] = '\0';
```

Comment:
Please use `sterm[sizeof(sterm) - 1] = '\0'`

Reply:
UT_LINESIZE is actually the limit of the length that a terminal device name can have.
I don't see nothing wrong here, maybe we could go down a bit on sterm[]'s size.

Reply:
Done. By the way, sterm's size of 32 is actually UT_LINESIZE.

Reply:
> Done. By the way, sterm's size of 32 is actually UT_LINESIZE.

Then initialize it using UT_LINESIZE instead of 32

Reply:
Already did.